src/component/handlers/composition/DraftEditorCompositionHandler.js

```diff
@@ -71,16 +78,18 @@ const DraftEditorCompositionHandler = {
    * twice could break the DOM, we only use the first event. Example: Arabic
    * Google Input Tools on Windows 8.1 fires `compositionend` three times.
    */
-  onCompositionEnd: function(editor: DraftEditor, e: SyntheticEvent<>): void {
+  onCompositionEnd: function(editor: DraftEditor): void {
     resolved = false;
     stillComposing = false;
     setTimeout(() => {
       if (!resolved) {
-        DraftEditorCompositionHandler.resolveComposition(editor, e);
+        DraftEditorCompositionHandler.resolveComposition(editor);
       }
     }, RESOLVE_DELAY);
   },
 
+  onSelect: editOnSelect,
```

Comment:
Adding this here fixes an issue where the selection wouldn't be properly updated during composition.
This would cause, for example, inline-styles to be not be applied during composition mode.